src/test/java/seedu/realodex/model/remark/RemarkTest.java

```diff
@@ -0,0 +1,61 @@
+package seedu.realodex.model.remark;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.realodex.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class RemarkTest {
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new Remark(null)); //Remark does not take in null,
+        // only empty string
+    }
+
```

Comment:
I think we are missing one test to be consistent with the other tests :

```suggestion
    @Test
    public void constructor_invalidRemark_throwsIllegalArgumentException() {
        String invalidRemark = "wh4tI$1nval1d/r/r /r /n";
        assertThrows(IllegalArgumentException.class, () -> new Remark(invalidRemark));
    }
```

Reply:
This test is not necessary, as there are no invalid remarks. All characters are accepted, and an empty remark will be recorded as empty.

Reply:
Agree with @appleraincoat

Reply:
I think an alternative to this would be to put a whitespace remark. In our implementation, the parser trims the remarks input so inputting `r/                    `  should still be an empty remark (it should already work i just tested). Perhaps this is something we should emphasise in the UG as well as test for just to be pedantic

Reply:
great suggestion! i have added the corresponding test. i have also slightly tweaked the implementation of the equals method to compare the trimmed versions.